src/tests/contract/routes/AccessTokenRoute.ts

```diff
@@ -0,0 +1,25 @@
+import express from "express";
+import asyncify from "express-asyncify";
+import { lambdaHandler } from "../../../AccessTokenHandler";
+import { APIGatewayProxyEvent } from "aws-lambda";
+import { translateHttpRequestToApiGatewayProxyEvent } from "../utils/ApiRequestUtils";
+import { Constants } from "../utils/Constants";
+
+process.env.SESSION_TABLE = "session-table";
+process.env.TXMA_QUEUE_URL = "txma-queue";
+process.env.ISSUER = "issuer";
+process.env.USE_MOCKED = "true";
+process.env.KMS_KEY_ARN = "kid";
+
+export const accessTokenRouter = asyncify(express.Router());
+
+// eslint-disable-next-line @typescript-eslint/no-misused-promises
```

Comment:
I tried removing the comment and resolve it, but was not able to solve it. If someone can help in here...